chapters/1-rationale.md

```diff
@@ -74,6 +74,7 @@ In an SPDX document, Relationship elements can be used to indicate relationships
 * YAML 1.2  see: https://yaml.org/spec/1.2/spec.html
 * JavaScript Object Notation (JSON) see: ECMA-404(https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf)
 * Resource Description Framework (RDF also referred to as RDF/XML) see: https://www.w3.org/TR/rdf-syntax-grammar/
+    The JSON Schema for SPDX can be found in the [SPDX Spec Git Repository Schema directory](https://github.com/spdx/spdx-spec/schemas/spdx-schema.json)
```

Comment:
Hi @goneall, content-wise this looks good, but putting it here looks like it's getting folded into the RDF bullet point when it renders. Can this maybe be moved one line up and made into a sub-bullet?